queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py

```diff
@@ -78,12 +79,23 @@ def correct_business_data(business: Business, correction_filing_rec: Filing,  #
         court_order_json = dpath.util.get(correction_filing, '/correction/courtOrder')
         filings.update_filing_court_order(correction_filing_rec, court_order_json)
 
-        # update business start date, if any is present
+    # update business start date, if any is present
     with suppress(IndexError, KeyError, TypeError):
         business_start_date = dpath.util.get(correction_filing, '/correction/startDate')
         if business_start_date:
             business.start_date = datetime.datetime.fromisoformat(business_start_date) + timedelta(hours=8)
 
+    # update share structure and resolutions, if any
+    with suppress(IndexError, KeyError, TypeError):
+        share_structure = dpath.util.get(correction_filing, '/correction/shareStructure')
+        shares.update_share_structure_ben_correction(business, share_structure)
```

Comment:
can we rename this to `update_share_structure_correction`?  Other legal types will have share structures as well.

Reply:
Yeah. Sure.